tests/security/audit/auditd.pm

```diff
@@ -0,0 +1,36 @@
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-Later
+#
+# Summary: Controlling the Auditd daemon as root by "systemctl" to verify it can work
+# Maintainer: llzhao <llzhao@suse.com>, shawnhao <weixuan.hao@suse.com>
+# Tags: poo#81772 tc#1768549
+
+use base 'opensusebasetest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run {
+    select_console 'root-console';
+
+    # Make sure audit packages are installed
+    zypper_call('in audit libaudit1');
+
+    # Check auditd status by default on a clean new VM
+    assert_script_run('systemctl status --no-pager auditd', sub { m/Active: active./ });
```

Comment:
I suggest using  systemctl('is-active auditd');

Reply:
Changed.